purchases/src/main/kotlin/com/revenuecat/purchases/common/FileHelper.kt

```diff
@@ -32,11 +32,11 @@ internal class FileHelper(
         return file.delete()
     }
 
-    fun readFilePerLines(filePath: String, dataListener: DataListener<Pair<String, Int>>) {
+    fun readFilePerLines(filePath: String, dataListener: DataListener<Pair<String, Int>>, maxLines: Int? = null) {
```

Comment:
This might be simpler without `null`:
```suggestion
    fun readFilePerLines(filePath: String, dataListener: DataListener<Pair<String, Int>>, maxLines: Int = Integer.MAX_VALUE) {
```